store/conf/attrs/zimbra-attrs.xml

```diff
@@ -5284,7 +5284,7 @@ TODO: delete them permanently from here
   </desc>
 </attr>
 
-<attr id="1133" name="zimbraMailDumpsterLifetime" type="duration" cardinality="single" optionalIn="account,cos" flags="accountInherited,domainAdminModifiable,accountInfo" since="7.0.0">
+<attr id="1133" name="zimbraMailDumpsterLifetime" type="duration" cardinality="single" optionalIn="globalConfig,account,cos" flags="accountInherited,domainAdminModifiable,accountInfo" since="7.0.0">
```

Comment:
add domain and domainInherited
```suggestion
<attr id="1133" name="zimbraMailDumpsterLifetime" type="duration" cardinality="single" optionalIn="globalConfig,domain,account,cos" flags="accountInherited,domainInherited,domainAdminModifiable,accountInfo" since="7.0.0">
```